source/chapters/user_interface.rst

```diff
@@ -10,42 +10,45 @@ You are going to want to get very familiar with this interface because it is
 the main way to use Mixxx. In this chapter, we present the default interface of
 Mixxx and describe its elements, knobs and faders.
 
-.. figure:: ../_static/2.2/overview.png
+.. figure:: ../_static/2.3/mixxx-interface.png
    :align: center
    :width: 100%
    :figwidth: 100%
-   :alt: The Mixxx interface - Deere skin
+   :alt: The Mixxx interface - Latenight skin
    :figclass: pretty-figures
 
-   The Mixxx interface - Deere skin
+   The Mixxx interface - Latenight skin
 
-This is the Deere skin. It is the default skin supplied with Mixxx 2.2. There
+This is the Latenight skin. It is the default skin supplied with Mixxx 2.3. There
 are a variety of other skins included with Mixxx. You should explore them all
-to figure out which one you prefer. This section will focus on Deere only.
+to figure out which one you prefer. This section will focus on Latenight only.
 
 .. _interface-skin-settings:
 
 Section Skin Settings
 =========================
 
-  .. figure:: ../_static/2.2/expansion-buttons-top.png
+  .. figure:: ../_static/2.3/skin-settings-top.png
    :align: center
-   :width: 430px
+   :width: 500px
    :figwidth: 90%
    :alt: The section expansion buttons
    :figclass: pretty-figures
 
    The GUI section quick toggles buttons in the top tool bar
 
-If you click on either :guilabel:`EFFECTS`, :guilabel:`SAMPLERS` or :guilabel:`MICS & AUX` then you will enable control sections for interacting with:
+If you click on either :guilabel:`WAVEFORMS`, :guilabel:`4 DECKS`, :guilabel:`MIXER`, :guilabel:`EFFECTS`, :guilabel:`SAMPLERS` or :guilabel:`MICS & AUX` then you will enable control sections for interacting with:
 
+* :ref:`interface-waveform`
+* :ref:`interface-decks`
+* :ref:`interface-mixer`
 * :ref:`interface-effects`
 * :ref:`interface-sampler`
 * :ref:`interface-mic`
 
 You can also use the specific :ref:`appendix-shortcuts`.
 
-  .. figure:: ../_static/2.2/expansion-buttons-skin-settings.png
+  .. figure:: ../_static/2.3/skin-settings-right.png
```

Comment:
Is 'Settings' a guilabel: ?

Reply:
In some skins it is, in others it's just a Gear icon ([U+2699](https://www.fileformat.info/info/unicode/char/2699/index.html), but IIRC we can't use Unicode chars because it breaks the PDF builds.

Reply:
Click the :guilabel:`SETTINGS` button or gear icon (depending on the Skin) to open the skin settings menu where you find common basic controls as well as some skin-specific options."